mappings/net/minecraft/server/world/ServerWorld.mapping

```diff
@@ -11,13 +12,18 @@ CLASS ur net/minecraft/server/world/ServerWorld
 	METHOD <init> (Lnet/minecraft/server/MinecraftServer;Ljava/util/concurrent/Executor;Lcjd;Lcjc;Lbti;Laff;Lvc;)V
 		ARG 1 server
 	METHOD B getDimensionalPersistentStateManager ()Lcjb;
+	METHOD D getForcedChunkIds ()Lit/unimi/dsi/fastutil/longs/LongSet;
```

Comment:
`getForcedChunks`

Reply:
Sure?
It actually returns only the indices of the corresponding array and not the real chunks

Reply:
Updated it

Reply:
For the record, it doesn't return "indices", it returns the hashes of the chunk xz.